### PR TITLE
Unit tests update: AdyenContext injection

### PR DESCRIPTION
**Pull request by nauaros**
## Summary

Update to inject the `AdyenContext` in our unit tests.

### CODE REVIEW

File: Tests/AdyenTests/Adyen Tests/Components/Boleto/BoletoComponentTests.swift [hunk 1]
```diff
@@ -292,19 +297,20 @@ class BoletoComponentTests: XCTestCase {
...
+                              adyenContext: Dummy.adyenContext,
                               configuration: mockConfiguration)
         sut.delegate = mockDelegate
         
         UIApplication.shared.keyWindow?.rootViewController = sut.viewController
         
         DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + .seconds(1)) {
-        
+
             let submitButton: SubmitButton? = self.sut.viewController.view.findView(by: "payButtonItem.button") as? SubmitButton
```
Comment (neelSharma12) [inline Tests/AdyenTests/Adyen Tests/Components/Boleto/BoletoComponentTests.swift:307]: why these extra space changes are coming? Do we need them?

**Review by neelSharma12 — APPROVED** [review]

File: Tests/AdyenTests/Adyen Tests/Components/Boleto/BoletoComponentTests.swift (hunk 1 above)
Comment (nauaros) [reply Tests/AdyenTests/Adyen Tests/Components/Boleto/BoletoComponentTests.swift:307]: nice catch, already removed them!

File: Tests/AdyenTests/Adyen Tests/Components/AdyenActionComponentTests.swift [hunk 2]
```diff
@@ -79,7 +79,7 @@ class AdyenActionComponentTests: XCTestCase {
     }
 
     func testAwaitAction() {
-        let sut = AdyenActionComponent(apiContext: Dummy.context)
+        let sut = AdyenActionComponent(apiContext: Dummy.context, adyenContext: Dummy.adyenContext)
```
Comment (descorp) [inline Tests/AdyenTests/Adyen Tests/Components/AdyenActionComponentTests.swift:82]: I thought `adyenContext ` should include `apiContext` ? 🤔

File: Tests/AdyenTests/Adyen Tests/DropIn/DropInActionTests.swift [hunk 3]
```diff
@@ -12,18 +12,26 @@ import XCTest
...
+    var adyenContext: AdyenContext!
     var sut: DropInComponent!
 
-    override func tearDown() {
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        adyenContext = Dummy.adyenContext
+    }
+
+    override func tearDownWithError() throws {
```
Comment (descorp) [inline Tests/AdyenTests/Adyen Tests/DropIn/DropInActionTests.swift:23]: I don't see telemetry tests here. 
Why do you need `setUp` and `tearDown` ?

**Review by descorp — APPROVED** [review]

**Pull request merged**